### PR TITLE
Update signals.rst to correct signal names.

### PR DESCRIPTION
**Pull request by MisoMosiSpy**
Fixes https://github.com/godotengine/godot-docs/issues/7048

1. Update signal names as per the coding conventions
2. Update images to reflect correct signal names (Also changed them from png to webp)
3. Added note about signal callback names

### CODE REVIEW

**Comment by Piralein** [comment]
There is already a PR for this: #7024 
(just for reference)

File: getting_started/step_by_step/signals.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:article_outdated: True
+:article_outdated: False
```
Comment (Piralein) [inline getting_started/step_by_step/signals.rst:1]: this line should be deleted and not set to false, as far as I know.

File: getting_started/step_by_step/signals.rst [hunk 2]
```diff
@@ -312,13 +312,17 @@ We can now connect the Timer to the Sprite2D in the ``_ready()`` function.
 
 The line reads like so: we connect the Timer's "timeout" signal to the node to
 which the script is attached. When the Timer emits ``timeout``, we want to call
-the function ``_on_Timer_timeout()``, that we need to define. Let's add it at the
+the function ``_on_timer_timeout()``, that we need to define. Let's add it at the
 bottom of our script and use it to toggle our sprite's visibility.
 
+.. note:: By convention, we name these callback methods in GDScript as 
+          "_on_node_name_signal_name" and in C# as "OnNodeNameSignaName".
```
Comment (Piralein) [inline getting_started/step_by_step/signals.rst:319]:
```suggestion
          "_on_node_name_signal_name" and in C# as "OnNodeNameSignalName".
```

**Review by Piralein — CHANGES_REQUESTED** [review]

File: getting_started/step_by_step/signals.rst (hunk 2 above)
Comment (MisoMosiSpy) [reply getting_started/step_by_step/signals.rst]: Thanks, updated as requested.

**Comment by MisoMosiSpy** [comment]
> There is already a PR for this: #7024 (just for reference)

Oops, my bad I should have checked. Anyways I have updated this PR to address all comments, let the maintainers pick any PR for merging.

Thanks for the review.

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Merging this PR since the other one has not been updated.

**Comment by Glukozavr** [comment]
Great guys! Sorry I got distracted with my pull-request, but glad it worked out anyway.

**Pull request merged**